src/main/webapp/app/course/manage/course-management.component.html

```diff
@@ -111,18 +111,6 @@ <h4 id="course-page-heading" jhiTranslate="artemisApp.course.home.title">Courses
                             <span *ngIf="course.presentationScore !== 0">{{ 'global.generic.yes' | translate }}</span>
                             <span *ngIf="course.presentationScore === 0">{{ 'global.generic.no' | translate }}</span>
                         </td>
-                        <td jhiOrionFilter [showInOrionWindow]="true">
```

Comment:
I think this was an accidental commit a while ago as this button is actually present within the 
`<div class="btn-group flex-btn-group-container">`

Reply:
please revert this change, this is a specific distinction for Orion, the IntelliJ plugin. I guess this button is shown in Orion (not in Artemis), the same one (below) is shown in Artemis, but not in Orion.